test/object-store/results.cpp

```diff
@@ -3525,3 +3525,88 @@ TEST_CASE("results: query helpers", "[include]") {
         CHECK(includes.get_description(table) == "INCLUDE(@links.class_linking_object.link)");
     }
 }
+
+TEST_CASE("notifications: objects with PK recreated") {
```

Comment:
This is a lot of duplicated code. The test should just be another section in the existing test case rather than copying and pasting all the setup.

Reply:
Actually, the setup is different.